public/form.yml

```diff
@@ -678,9 +679,9 @@ definitions:
             color: '#4FAB5B'
         - name:
             en: |-
-              I can't work, or my hours are reduced, because I'm advised to by a doctor or public health authority to quarantine.
+              I can’t work because I’m advised by a doctor or public health authority to quarantine.
```

Comment:
Probably not a big deal, don't know if it's deliberate, but pointing it out. Using ```’``` instead of ```'```.

Reply:
This is probably from copying directly from the Google doc.